### DISCUSSION TITLE
Publish binaries as GitHub releases

### DISCUSSION THREAD

**Discussion by thecodrr** [c-smile/sciter-js-sdk#269] [General]
Since sciter-js is already on GitHub, it would be amazing if the binaries were uploaded to GitHub Releases for easier downloading & trying. Currently, the only way to download Sciter is to clone the whole repo which is in GBs and includes a lot of unnecessary stuff. The second option is to use a GitHub folder downloading tool — even then you are left with a lot of frustration.

It would be really helpful for developers if releases were versioned along with their changelogs like you are already doing in the logfile.

**Comment by pravic** [comment]
Actually, this can be automated with Github actions based on the commit message.

**Comment by 8ctopus** [comment]
another option is `git clone --depth 1` which will only clone the repo to a depth of 1.

**Comment by thecodrr** [comment]
> @8ctopus another option is git clone --depth 1 which will only clone the repo to a depth of 1.

Still not the same as just downloading the binaries which is the standard everywhere.

**Comment by c-smile** [comment]
Binaries without corresponding headers are barely useful.

**Comment by thecodrr** [comment]
> Binaries without corresponding headers are barely useful.

I meant the executables. But you could also pack the headers in a tar.gz/zip file and upload that along with everything else.

**Comment by c-smile** [comment]
documentation + headers + executables are pretty much all what changes. So it is not clear what shall be put in that zip file and how it shall be different from https://github.com/c-smile/sciter-js-sdk/archive/refs/heads/main.zip

**Reply by thecodrr** [reply]
> So it is not clear what shall be put in that zip file and how it shall be different from https://github.com/c-smile/sciter-js-sdk/archive/refs/heads/main.zip

That zip file contains _everything_. Currently, it is not possible to easily download only the things you want like it is normally. For example, I am only concerned with `scapp`, `usciter` & `inspector` and having those available in a versioned downloadable zip file is would be a huge help.
